packages/record-tuple-polyfill/src/index.js

```diff
@@ -16,3 +16,4 @@
 
 export { Record } from "./record";
 export { Tuple } from "./tuple";
+export { parseImmutable } from "./json";
```

Comment:
Is it ok if I export it like this instead of mutating the JSON object?

Reply:
Yes, I think that's preferable.